### PR TITLE
Update for LGR_OPTIONS bit function

### PR DESCRIPTION
**Pull request by Hwurzburg**

### CODE REVIEW

**Comment by rmackay9** [comment]
Thanks for this.  It's a bit wordy perhaps but looks OK.

**Comment by Hwurzburg** [comment]
reworded....as compact as I can make it and still have all the info buried...

**Comment by rmackay9** [comment]
Great, thanks!!

**Comment by Hwurzburg** [comment]
@rmackay9 what about the question? question: if you enter and exit Copter LAND before actually landing is completed, will the gear automatically retract?

**Comment by rmackay9** [comment]
@Hwurzburg, no I don't think it will automatically retract.  It will only retract with the takeoff is complete I think.
